project/src/const/app-route.ts

```diff
@@ -0,0 +1,8 @@
+export enum AppRoute {
+  MAIN = '/',
+  SIGN_IN = '/login',
+  MY_LIST = 'myList',
```

Comment:
```suggestion
  MY_LIST = 'my-list',
```
URL не учитывает разницу между строчными и прописными.